bundles/AdminBundle/Resources/public/js/pimcore/asset/folder.js

```diff
@@ -62,6 +62,11 @@ pimcore.asset.folder = Class.create(pimcore.asset.asset, {
             fields: ['url', "filename", "filenameDisplay", "type", "id", "idPath"],
             listeners: {
                 "load": function () {
+                    if(this.store.getCount() === 0) {
+                        this.tabbar.setActiveItem(this.listfolder.getLayout());
+                        this.tabbar.remove(this.dataview);
```

Comment:
Great, it works and is ready to merge.

it will also work with `this.tabbar.remove(this.dataview);` alone, since it will just "fallback" to `listfolder` (because it is the next one), if `dataview` is not present.
So `this.tabbar.setActiveItem(this.listfolder.getLayout());` is not strictly necessary, wondering if it would be worth to be removed (because it's obsolete at the moment) or just kept it (more clear and future-proof). wdyt?

Reply:
yes, sounds right to me, i am merging it. Thank you again for the PR and for clarifyng